### ISSUE TITLE
Quicksilver is spontaneously appearing, often when no keys have been pressed

### ISSUE THREAD

**Issue by robacarp** [quicksilver/Quicksilver#2117]
I poked around in my settings and can't find anything that changed recently, but the quicksilver bezel is popping up randomly when I'm doing a variety of things. Most recently I was hands-off keyboard and had been exclusively mousing for several minutes. I believe I've been witnessing the behavior since 1.3.0 and is persisting in 1.3.1.

The Activation settings haven't changed (in years!) and are as follows: 

![](http://robacarp.com/images/skitchd/2015.10.06.10.13.05-Preferences.png)

Application is set to Start at login, don't show in dock or menu bar. QS is up to date as are all plugins. Loaded plugins are:
- bezel interface
- calculator plugin
- clipboard plugin
- keyboard triggers
- primer interface
- shelf plugin

Searching through the system Console, quicksilver has been reporting: "Quicksilver: bad data for public.utf16-external-plain-text", but the timestamps don't seem to line up with when I'm noticing spontaneous activations.

I poked around the web and the github issues and didn't see anything related to unexpected activation, though I spotted several on failed activation. 

Currently on OSX 10.10.5.

I'm happy to help debug the issue or provide any additional information if needed!

Thanks QS!

**Comment by skurfer** [comment]
Someone else just [reported the same thing][1], but we never figured it out. What triggers do you have set up?

[1]: https://groups.google.com/forum/#!topic/blacktree-quicksilver/S3Sv0qKILdQ

**Comment by robacarp** [comment]
@skurfer The only trigger I have is the Apple-Esc shortcut. I have some shortcuts that, I think, are from the minimizer plugin (Minimize All, Restore All, Restore First/Last). I don't have that plugin enabled anymore, and I'm not 100% sure it ever installed correctly. 

![screen shot 2015-10-09 at 10 23 59](https://cloud.githubusercontent.com/assets/208647/10399427/e7cfa42a-6e6f-11e5-8df0-480d4f2abdc8.png)

I upgraded to OSX10.11 yesterday and this morning I saw the same unexpected activation.
